src/elastio-snap.c

```diff
@@ -1970,6 +2014,7 @@ static int __file_unlink(struct file *filp, int close, int force){
 		goto mnt_error;
 	}
 
+	file_unlock(filp);
```

Comment:
Why do we need to call `file_unlock(filp);` 2 times? Once is here, before `vfs_unlink` call and then again after.

Reply:
There's a lock, not unlock :) 
The sequence is:
1. unlock
2. do whatever we want with vfs_* functions
3. lock again

Reply:
exactly! I overlooked )